projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/aux-key.e2e-spec.ts

```diff
@@ -0,0 +1,270 @@
+import { testProductUrl } from '../../helpers/accessibility/tabbing-order';
+
+context('Auxiliary Keys', () => {
+  describe('Category Navigation', () => {
+    before(() => {
+      cy.server();
+      cy.route(
+        `${Cypress.env('API_URL')}/rest/v2/electronics-spa/cms/components*`
+      ).as('getComponents');
+      cy.visit('/');
+      cy.wait('@getComponents');
+    });
+
+    it('should open menu with space key', () => {
+      cy.get('cx-page-slot[ng-reflect-position="NavigationBar"]').within(() => {
+        cy.get('cx-navigation-ui')
+          .find('nav')
+          .should('have.length', 30);
+        cy.get('cx-navigation-ui nav h5')
+          .contains('Brands')
+          .should('be.visible');
+        cy.wait(1000); // TODO: Wait stabilizes test, change after cx-navigation-ui refactor
+        cy.get('cx-navigation-ui nav span')
+          .first()
+          .focus();
+        cy.focused().trigger('keydown', {
+          key: ' ',
+          code: 'Space',
+          force: true,
+        });
+        cy.get('cx-navigation-ui nav div[class="wrapper"]').should(
+          'be.visible'
+        );
+      });
+    });
+
+    it('should tab through menu items', () => {
```

Comment:
I think we don't need to check if we are able to tab through the elements in aux keys tests

Reply:
also applies to other lines of this file